core-library/src/main/java/org/silverpeas/core/admin/service/DefaultOrganizationController.java

```diff
@@ -1174,4 +1138,49 @@ public List<SpaceInstLight> getPathToComponent(String componentId) {
     }
     return new ArrayList<>();
   }
+
+  private UserDetail[] removeSensitiveData(final UserDetail[] userDetails) {
+    User requester = User.getCurrentRequester();
+    List<UserDetail> results = removeSensitiveData(List.of(userDetails), requester);
+    return results.toArray(new UserDetail[0]);
+  }
+
+  private <T extends UserDetail> List<T> removeSensitiveData(final List<T> users,
+      final User requester) {
+    if (!users.isEmpty() && requester != null) {
+      return users.stream()
```

Comment:
The given list could be a `SilverpeasList` one. So, the returned list MUST not be a new list instance. The list has just to be walked to apply on each User instance the sensitive data processing.

Reply:
Yes it is done in my current fix (not yet pushed as I am resolving other issues)